CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.0.0-beta10 (in progress)
+==========================
+
+New Features
+------------
+
+- By default, twilio-video.js connects to your nearest signaling server, as determined by
+  [latency based routing](https://www.twilio.com/docs/video/ip-address-whitelisting#signaling-communication).
+  You can now override this behavior by using a new ConnectOptions flag called `region`. This will make
+  sure that your signaling traffic will terminate in the specified region. (JSDK-2338)
+
+  ```js
+  const { connect } = require('twilio-video');
+  const room = await connect(token, {
+    region: 'de1'
+  });
+  ```
+
+  This will guarantee that your signaling traffic will terminate in Germany. For other possible values
+  for region, please refer to this [table](https://www.twilio.com/docs/video/ip-address-whitelisting#media-servers).
```

Comment:
Link is pointing to media-servers. shouldn't it point to https://www.twilio.com/docs/video/ip-address-whitelisting#signaling-communication ?

Reply:
@syerrapragada I think at the time of implementing region selection, only the media servers section contained the list of all possible regions. I will make this change. Thanks for catching it.